src/main.rs

```diff
@@ -197,6 +197,7 @@ async fn mempool_task<Enforcer, RpcClient, F, Fut>(
             Ok(res) => res,
             Err(err) => {
                 let err = anyhow::Error::from(err);
+                tracing::error!("mempool sync error: {err:#}");
```

Comment:
This error will be logged by the error handler - if you want to make it explicit that it is a mempool sync error, either do
```
let err = anyhow::anyhow!("mempool sync error: {err:#}")
```

or use a wrapper error struct that has this message wrapper, around an inner enum for the error cases